packages/documentation-site/patternfly-docs/content/design-guidelines/styles/icons/icons.js

```diff
@@ -1244,6 +1237,55 @@ export const iconsData = [
     "Type": "Object",
     "Contextual_usage": "Indicates availability of a catalog/library to browse",
   },
+  {
+    "Style": "pficon",
+    "Name": "pficon-critical-risk",
+    "React_name": "CriticalRiskIcon",
+    "Type": "Status",
+    "Contextual_usage": "Indicates critical severity for an error or issue. This is an old icon, going forward, use the pficon-severity-critical icon.",
+  },
+  {
+    "Style": "pficon",
+    "Name": "pficon-severity-critical",
+    "React_name": "SeverityCriticalIcon",
+    "Type": "Status",
+    "Contextual_usage": "Indicates critical severity for an error or issue. This icon should be used instead of pficon-severity-critical for new implementations.",
```

Comment:
```suggestion
    "Contextual_usage": "Indicates critical severity for an error or issue. This icon should be used instead of pficon-critical-risk for new implementations.",
```
typo